homeassistant/components/panasonic_viera/__init__.py

```diff
@@ -80,15 +79,12 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
     host = config[CONF_HOST]
     port = config[CONF_PORT]
 
-    if (on_action := config[CONF_ON_ACTION]) is not None:
-        on_action = Script(hass, on_action, config[CONF_NAME], DOMAIN)
```

Comment:
Shouldn't we create an issue registry issue saying this is no longer supported and explaining what do do instead?
How can the user remove the `on_action` from their config entry data?